docs/userguide/icu4c-readme.md

```diff
@@ -412,121 +410,146 @@ Depending on the platform and the type of installation, we recommend a small num
+> :point_right: **Note**: Using older versions of the MSVC compiler is generally not recommended due to the improved support for the C++11 standard in newer versions of the compiler.
 
-*   Open the "Developer Command Prompt" shortcut from the Start Menu. (This will open up a new command line window).
-*   From within the "Developer Command Prompt" change directory (<tt>cd</tt>) to the ICU source directory.
-*   You can then use either '<tt>msbuild</tt>' directly, or you can use the '<tt>devenv.com</tt>' command to build ICU.
-*   Using <tt>MSBUILD</tt>:
+#### Re-targeting the Windows 10 SDK for the UWP projects
 
-*   To build the 32-bit Debug version, use the following command line:  
-    `'msbuild source\allinone\allinone.sln /p:Configuration=Debug /p:Platform=Win32'`.
-*   To build the 64-bit Release version, use the following command line:  
-    `'msbuild source\allinone\allinone.sln /p:Configuration=Release /p:Platform=x64'`.
+If the version of the Windows 10 SDK that you have installed does not match the version used by the UWP projects, then you will need to "retarget" them to use the version of the SDK that you have installed instead. There are two ways to do this:
+
+* In Visual Studio you can right-click on the UWP projects in the 'Solution Explorer' and select the option 'Retarget Projects' from the context menu. This will open up a window where you can select the SDK version to target from a drop-down list of the various SDKs that are installed on the machine.
+* Alternatively, you can manually edit the shared file called `Build.Windows.UWP.ProjectConfiguration.props` which is located in the `allinone` directory. You will need to change the of the `WindowsTargetPlatformVersion` property to the version of the SDK that you would like to use instead.
 
-*   Using <tt>devenv.com</tt>:
+#### Using MSBUILD At The Command Line
 
-*   To build the 32-bit Debug version, use the following command line:  
+You can build ICU from the command line instead of using the Visual Studio GUI. Assuming that you have properly installed Visual Studio to support command line building, you should have a shortcut for the "Developer Command Prompt" listed in the Start Menu. (For Visual Studio 2017 you will need to install the "Desktop development with C++" option).
+
+* Open the "Developer Command Prompt" shortcut from the Start Menu. (This will open up a new command line window).
+* From within the "Developer Command Prompt" change directory (`cd`) to the ICU source directory.
+* You can then use either `msbuild` directly, or you can use the `devenv.com` command to build ICU.
+* Using `MSBUILD`:
+  - To build the 32-bit Debug version, use the following command line:
+    `'msbuild source\allinone\allinone.sln /p:Configuration=Debug /p:Platform=Win32'`.
```

Comment:
```suggestion
    <br/>`'msbuild source\allinone\allinone.sln /p:Configuration=Debug /p:Platform=Win32'`.
```

For some reason the rendered markdown page doesn't seem to keep the line breaks for these... :/
I wonder if maybe we could add <br/> before the command line example to force it to break?
Or maybe a backslash would be better for Markdown?

Reply:
Thinking on forcing the newline for this inline code, it seems more idiomatic in Markdown to use a code fence to ensure the newline and prominence.  So I changed the inline code to a code fence and removed the single quotes at the beginning and end that seemed redundant at this point. How does that sound?

Reply:
Thanks. That looks much better in the formatted output on the preview: 
https://echeran.github.io/icu/userguide/icu4c-readme.html
Thanks!